src/app/components/shell/router.js

```diff
@@ -159,6 +159,8 @@ function ImportedPage({name}) {
 
     useAnalyticsPageView();
 
+    useEffect(() => window.scrollTo(0, 0), [name]);
+
```

Comment:
I think doing this will also scroll to the top when you hit the back button.  In that case you probably want to leave the page scrolled to it's original position.  To do so you could use the stateObject parameter of push/replace state to store the scroll position, then restore it later.

That's assuming you have control over the push/replace State call, not sure if you do.

Reply:
Good thinking. React Router does have a solution. :)

Reply:
Or, actually, it doesn't. :(